docs/component-spec/modules.md

```diff
@@ -99,6 +99,8 @@ When new versions of components are released, updates may be needed to component
 * If the release is a new **minor** version, the component developer *should* notify maintainers of all components and products listed as dependents in the Origami registry, immediately after the release.
 * If the release is a new **patch** version, no notifications need be sent.
 
+The first released version of a module *must* be `v1.0.0`.  Versions lower than 1 are subject to different semver parsing logic, which is a nuance best avoided.
```

Comment:
v prefix is actually considered best practice, I'm told by @matthew-andrews.  I suggest we migrate *to* a standard of using the v prefix.